<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgMeals</class>
 <widget class="QWidget" name="wdgMeals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>669</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Your meals</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCalendarWidget" name="calendar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalHeaderFormat">
          <enum>QCalendarWidget::NoVerticalHeader</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_8">
        <attribute name="icon">
         <iconset resource="../images/caloriestracker.qrc">
          <normaloff>:/caloriestracker/meals.png</normaloff>:/caloriestracker/meals.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Meals</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="mqtwObjects" name="tblMeals" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="lblFound">
             <property name="text">
              <string>Registers found</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="../images/caloriestracker.qrc">
          <normaloff>:/caloriestracker/pie.png</normaloff>:/caloriestracker/pie.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Meals chart</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="VCPie" name="wdgPieMeals" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="../images/caloriestracker.qrc">
          <normaloff>:/caloriestracker/pie.png</normaloff>:/caloriestracker/pie.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Food types chart</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="VCPie" name="wdgPieFoodtypes" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset resource="../images/caloriestracker.qrc">
          <normaloff>:/caloriestracker/investment.png</normaloff>:/caloriestracker/investment.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Daily evolution</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="VCTemporalSeries" name="wdgTS" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMealNew">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/add.png</normaloff>:/caloriestracker/add.png</iconset>
   </property>
   <property name="text">
    <string>New meal</string>
   </property>
   <property name="toolTip">
    <string>New meal</string>
   </property>
  </action>
  <action name="actionMealDelete">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/list-remove.png</normaloff>:/caloriestracker/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete meal</string>
   </property>
   <property name="toolTip">
    <string>Delete meal</string>
   </property>
  </action>
  <action name="actionMealEdit">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/document-edit.png</normaloff>:/caloriestracker/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit meal</string>
   </property>
   <property name="toolTip">
    <string>Edit meal</string>
   </property>
  </action>
  <action name="actionProductEdit">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/books.png</normaloff>:/caloriestracker/books.png</iconset>
   </property>
   <property name="text">
    <string>Edit product</string>
   </property>
   <property name="toolTip">
    <string>Edit product</string>
   </property>
  </action>
  <action name="actionMealDeleteDay">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/button_cancel.png</normaloff>:/caloriestracker/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete selected date meals</string>
   </property>
   <property name="toolTip">
    <string>Delete selected date meals</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VCPie</class>
   <extends>QWidget</extends>
   <header location="global">myqcharts</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VCTemporalSeries</class>
   <extends>QWidget</extends>
   <header location="global">myqcharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/caloriestracker.qrc"/>
 </resources>
 <connections/>
</ui>
